apps/portal/src/app/connect/wallet/overview/page.mdx

```diff
@@ -0,0 +1,97 @@
+import { Callout, DocImage, createMetadata, FeatureCard } from "@doc";
+import customModal from "./assets/custom-modals.png";
+import EcosystemWalletOverview from "./assets/ecosystem-wallet.png";
+
+export const metadata = createMetadata({
+	image: {
+		title: "thirdweb Wallets",
+		icon: "wallets",
+	},
+	title: "thirdweb Wallets | thirdweb",
+	description:
+		"thirdweb wallets provide a comprehensive toolkit for connecting users to applications and games. We offer a way to let users sign in however you want. Be it, connecting an external wallet, or creating a seamless login experiences for brand new users, or enabling you to own your web3 identity stack, we have a solution for you.",
+});
+
+# Overview
+
+A flexible sign-up flow that accommodates different preferences is critical when onboarding users. thirdweb offers multiple wallet solutions to meet various integration needs:
+
+1. **Connect** - Traditional wallet connection with 350+ providers
+2. **In-App Wallet** - Seamless onboarding with email, social, and passkey options
+3. **Ecosystem Wallet** - Branded wallet service for cross-application authentication
+
+<DocImage src={EcosystemWalletOverview} />
+
+### Features
+
+<div
+	className="my-4 grid gap-2 md:grid-cols-2 lg:grid-cols-2 "
+>
+	<FeatureCard
+		title="Wallet Support"
+		description="350+ wallet providers supported"
+		iconUrl="/icons/feature-cards/wallet.svg"
+	/>
+    <FeatureCard
+    	title="Fiat On-Ramp"
+    	description="Enable purchase options directly in modal"
+        iconUrl="/icons/feature-cards/fiat-ramp.svg"
+    />
+
+    <FeatureCard
+    	title="Seamless onboarding"
+    	description="Email, phone, social, and passkey options"
+        iconUrl="/icons/feature-cards/fingerprint.svg"
+    />
+    <FeatureCard
+		title="Customizable"
+		description="Match your applications branding"
+		iconUrl="/icons/feature-cards/customize.svg"
+	/>
+		<FeatureCard
+		title="Gasless"
+		description="Enable free transactions with smart accounts"
+		iconUrl="/icons/feature-cards/gasless.svg"
+	/>
+
+    <FeatureCard
+    	title="Localization"
+    	description="Support for multiple languages"
+        iconUrl="/icons/feature-cards/localization.svg"
+    />
+	<FeatureCard
+		title="Cross-Chain Support"
+		description="Enable end users to view assets, balances, and activities for apps and games across any EVM-compatible chain"
+		iconUrl="/icons/feature-cards/compatibility.svg"
+	/>
+	<FeatureCard
+		title="User Attribution & Analytics"
+		description="Unlock analytics for developers and end user activity across your ecosystem"
+		iconUrl="/icons/feature-cards/compatibility.svg"
+	/>
+</div>
+
+### Solutions
+
+| SOLUTION | BEST FOR | PRICING |
+|----------|----------|----------|
+| Connect | Applications needing traditional EOA wallet integration | Free |
+| In-App Wallet | Mainstream applications requiring seamless onboarding | [Free up to 1,000 MAW, then $0.02/MAW](https://thirdweb.com/pricing) |
```

Comment:
Only 1,000? I thought it was a lot more than that? Also I think it's just wallets not monthly active, right?